roles/backup/templates/backup-content-k8s-job.yaml.j2

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ ansible_operator_meta.name }}-backup-content
+  namespace: {{ backup_pvc_namespace }}
+  labels:
+    app.kubernetes.io/name: '{{ deployment_type }}-backup-content'
+    app.kubernetes.io/instance: '{{ deployment_type }}-backup-content-{{ ansible_operator_meta.name }}'
+    app.kubernetes.io/component: backup-content
+    app.kubernetes.io/part-of: '{{ deployment_type }}'
+    app.kubernetes.io/managed-by: '{{ deployment_type }}-operator'
+spec:
+  ttlSecondsAfterFinished: 60
+  template:
+    metadata:
+      annotations:
+    spec:
+{% if is_k8s %}
+      securityContext:
+        runAsUser: 1000
+{% endif %}
+      containers:
+      - name: backup-content
+        image: quay.io/ansible/galaxy-ng:latest # TODO: Add set image tasks, and var ref here, etc.
```

Comment:
Do we want to switch this value to the correct variable name in this PR ?
Also do we need to handle resource limit on this container ?

Reply:
Agreed, thanks for calling these points out!  I just addressed both in the commit I just pushed.